### ISSUE TITLE
How to test SP-Nas after training ?

### ISSUE THREAD

**Issue by AyeshaNirma** [huawei-noah/vega#26]
I am sorry but it is puzzling on how to test Sp-Nas after training finishes. So my problem is that I have trained SP-Net in the `example` folder using the following command: 
`python run_example.py ./nas/sp_nas/spnas.yml` 

so the code is using the config file that you guys provide called `/nas/sp_nas/faster_rcnn_r50_fpn_1x.py` . It trains fine and during training it evaluates on the validation set and everything is fine and mAp is also reasonable.
Without changing anything in the code or the .yml file or anything, basically cloning it and setting up the dataset paths and pre-trained model and thats it. 

However, after training finishes I am trying to run the saved model in the folder: 
`examples/tasks/0719.042952.773/output/2/1112-1112-11111-21-1-11.pth`

Using the command 
`python test.py vega/examples/nas/sp_nas/faster_rcnn_r50_fpn_1x.py --checkpoint examples/tasks/0719.042952.773/output/2/1112-1112-11111-21-1-11.pth --out res.pkl`
but it gives basically 0 mAp which I know is wrong. Do I need to change something in the config file `faster_rcnn_r50_fpn_1x.py` or how can I run the test, could you please elaborate ?

**Comment by hasanirtiza** [comment]
You do have a config.json file in the folder `examples/tasks/zzz.xxxx/workers/nas1/3/config.json'. This is you network architecture file in mmdetection style.

**Comment by AyeshaNirma** [comment]
Thank u so much ... it works !!!!

**Comment by fhaji75** [comment]
> I am sorry but it is puzzling on how to test Sp-Nas after training finishes. So my problem is that I have trained SP-Net in the `example` folder using the following command: `python run_example.py ./nas/sp_nas/spnas.yml`
> 
> so the code is using the config file that you guys provide called `/nas/sp_nas/faster_rcnn_r50_fpn_1x.py` . It trains fine and during training it evaluates on the validation set and everything is fine and mAp is also reasonable. Without changing anything in the code or the .yml file or anything, basically cloning it and setting up the dataset paths and pre-trained model and thats it.
> 
> However, after training finishes I am trying to run the saved model in the folder: `examples/tasks/0719.042952.773/output/2/1112-1112-11111-21-1-11.pth`
> 
> Using the command `python test.py vega/examples/nas/sp_nas/faster_rcnn_r50_fpn_1x.py --checkpoint examples/tasks/0719.042952.773/output/2/1112-1112-11111-21-1-11.pth --out res.pkl` but it gives basically 0 mAp which I know is wrong. Do I need to change something in the config file `faster_rcnn_r50_fpn_1x.py` or how can I run the test, could you please elaborate ?

Hello, I couldn't find run_example.py script in this repo, Where is that?
